books/helpers/rest_helpers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.response import Response
+
+
+def make_formatted_response(status_code, data=None, message=None, error=None):
+    if error:
+        formatted_response = {'status_code': status_code,
+                              'status': 'false',
+                              'error': error
+                              }
+
+        return Response(formatted_response, status=status_code)
+    else:
+        if data is None:
+            data = []
+        formatted_response = {'status_code': status_code,
+                              'status': 'success',
+                              'data': data
+                              }
+        if message is not None:
+            formatted_response['message'] = message
+
+        return Response(formatted_response, status=status_code)
```

Comment:
@xhahid43eb false and success in status doesn't make sense, one is boolean other one is a string. Method name should be simple yet elaborative. Here is how I will do this. So you only need to send response status other than 200

```suggestion
def format_response(data, message=None, error=None, status_code=status.HTTP_200_OK):
    if error:
        formatted_response = {'status_code': status_code, 'status': 'failure', 'error': error}
    else:
        formatted_response = {'data':data or [], 'status_code': status_code, 'status': 'success'}
        if message:
            formatted_response['message'] = message
    return Response(formatted_response, status=status_code)
```

Reply:
My bad I will fix it.